wpical/src/main/native/include/fieldmap.h

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <numbers>
+#include <vector>
+
+#include <tagpose.h>
+#include <wpi/json.h>
+
+class fieldmap {
+ public:
+  fieldmap();
+  explicit fieldmap(wpi::json map);
+  tag::pose getTag(int tag);
+  int getNumTags();
+  double minimizeAngle(double angle);
```

Comment:
See comments in fieldmap.cpp.  fieldmap(), getTag(), and getNumTags() can just be inlined here.